docstore/memdocstore/mem.go

```diff
@@ -191,7 +191,7 @@ func (c *collection) RunActions(ctx context.Context, actions []*driver.Action, o
 		}
 	}
 
-	beforeGets, gets, writes, afterGets := driver.GroupActions(actions)
+	beforeGets, gets, writes, _, afterGets := driver.GroupActions(actions)
```

Comment:
You need to implement atomic writes for this, mongo and firestore.

Reply:
yes, as we discussed planned, we will be delivering it immediately after aws and driver is in. We will do the azure, gco and mongo
https://github.com/google/go-cloud/issues/3501#issuecomment-2435614003

Reply:
OK, thanks, I forgot.